### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by cueavyqwp** [pypi/support#3756]
### PyPI Username

`cueavyqwp`

### Reason for request

Lost 2FA authentication app and recovery codes
Can access to email address
Still have the security token

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by ewdurbin** [comment]
User `cueavyqwp` has valid recovery codes generated as of 2023-09-27 10:34:10.326325.

Please use those recovery codes to regain access to your account.

<span style="font-size: smaller;"><i>Note: This is an automated reply. In general recovery requests where valid recovery codes exist will be processed 3-4 weeks after this reply has been sent.</i></span>

**Comment by cueavyqwp** [comment]
> 用户 has valid recovery codes generated as of 2023-09-27 10:34:10.326325.`cueavyqwp`
> 
> Please use those recovery codes to regain access to your account.
> 
> Note: This is an automated reply. In general recovery requests where valid recovery codes exist will be processed 3-4 weeks after this reply has been sent.

but I lost my recovery codes

**Comment by cueavyqwp** [comment]
I initially set up two-factor authentication (2FA) and generated recovery codes.

I saved the recovery codes in a zip file protected with a password.

Subsequently, I transferred the file to my 2FA device, without saving it on my PC.

I typically use tokens to publish Python modules, so I've never relied on 2FA.

After a considerable period, I lost my 2FA device, necessitating the setup of a new 2FA.

Unfortunately, I forgot to save my recovery codes.

Hence, I initiated an account recovery request.

This request has been open for a month now, with only an automated response received. Is there anyone who can assist me?
